modules/tensor_mechanics/src/postprocessors/TorqueReaction.C

```diff
@@ -21,7 +21,7 @@ TorqueReaction::validParams()
   InputParameters params = NodalPostprocessor::validParams();
   params.addClassDescription("TorqueReaction calculates the torque in 2D and 3D"
                              "about a user-specified axis of rotation centered"
-                             "at a user-specied origin.");
+                             "at a user-specified origin.");
   params.addRequiredParam<std::vector<AuxVariableName>>("reaction_force_variables",
                                                         "The reaction variables");
   params.addParam<RealVectorValue>(
```

Comment:
Maybe we should make the parameter names in this class consistent with the new ones. I don't think it's widely used -- we just made it for a regression test.

Reply:
I think I prefer 'origin' and 'direction' like you have in the new code. At least they're both specifying the same quantities, although with different names now. We could go back later and update this.